libpromises/evalfunction.c

```diff
@@ -7876,6 +7876,7 @@ void ModuleProtocol(EvalContext *ctx, char *command, const char *line, int print
         if (CheckID(content))
         {
             Buffer *tagbuf = StringSetToBuffer(tags, ',');
+            CanonifyNameInPlace(content);
```

Comment:
I think we need to add some verbose logging here as well because otherwise the above log message may be confusing/misleading if canonification (?) happens afterwards.

Reply:
OK.

Reply:
Logging added